reposerver/repository/repository.go

```diff
@@ -1303,6 +1303,9 @@ func (s *Service) newHelmClientResolveRevision(repo *v1alpha1.Repository, revisi
 	if helm.IsVersion(revision) {
 		return helmClient, revision, nil
 	}
+	if repo.EnableOCI {
+		return nil, "", errors.New("OCI helm registers don't support semver ranges. Exact revision must be specified.")
+	}
```

Comment:
Is this documented yet?

Reply:
Looks like we don't have documentation about OCI registries. Adding it in next PR and will mention semver restriction.